AirSkyBoat/AirSkyBoat
# NM Spawn points and persistence update

**_I affirm:_**

## What does this pull request do?
+ Fixes Bright Handed Kunberry persistence
+ Fixes missing UpdateNMSpawnPoint on persistence
+ Adds missing nm spawn points
+ Fixes missing ENABLE_WOTG on Toxic Tamlyn
Fixes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/941


## Steps to test these changes
!spawnmob 17285526
!gotoid 17285526
Kill Bright Handed Kunberry
Restart server
See no NM
